content/en/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: Update API Objects in Place Using kubectl patch
+title: Update API Objects in place using kubectl patch
```

Comment:
The original title matches the style convention of existing pages (initial capitalization)

Reply:
looks like certain words (in, a, an, for, with, the) are not capitalized